lms/djangoapps/courseware/docs/decisions/0004-personalized-relative-dates.rst

```diff
@@ -0,0 +1,71 @@
+0004. Personalized/Relative Dates in Courseware
+***********************************************
+
+~~~~~~~
+Context
+~~~~~~~
+
+Self-paced courses are, based on the experience of users on edx.org,
+less engaging and less likely to be completed, than instructor-paced courses.
+Research shows that learners are better able to complete a course of study
+when that course gives them multiple incremental deadlines along the way,
+rather than one single deadline at the end of the course. Unfortunately,
+self-paced courses are currently only able to offer the latter experience.
+
+``edx-when`` is a library currently used by ``edx-platform`` to provide a
+read-optimized view of the dates associated with course content. It is updated
+when a course is published, and stores dates in a dedicated table in the
+relational database.
+
+~~~~~~~~
+Decision
+~~~~~~~~
+
+In order to support incremental deadlines in self-paced courses, we want
+to extend edx-platform with the following capabilities:
+
+1.  Extend the usage of edx-when in the LMS to begin reading the ``rel_date``
+    field, and convert it when loading by interpreting it as an offset
+    from the users ``start`` as defined by their ``Schedule``.
+2.  When a self-paced course is published, add relative due dates to it,
```

Comment:
am I correct in thinking that each mention of 'relative dates' and 'relative due dates' in this doc should actually be 'date offsets'